multidim-interop/versions.ts

```diff
@@ -98,6 +108,21 @@ export const versions: Array<Version> = [
         secureChannels: ["noise"],
         muxers: ["mplex", "yamux"],
     },
+    // TODO enable webrtc direct for v0.45
```

Comment:
What is the blocker here? We do support it for `js-libp2p` `v0.44`.

https://github.com/libp2p/test-plans/blob/4a3553280826eb1386b67e183aa521b27ef7ac83/multidim-interop/versions.ts#L94-L100